src/tools/moveedge.cpp

```diff
@@ -1,32 +1,76 @@
 #include "moveedge.h"
 #include "editorview.h"
 #include "edgeitem.h"
+#include "gripitem.h"
 #include <QMouseEvent>
 
 MoveEdgeTool::MoveEdgeTool()
 : BaseTool()
 {
 }
 
+void MoveEdgeTool::activated(EditorView* editor)
+{
+  QList<GripItem*> grips = editor->itemsOfType<GripItem>();
+  for (GripItem* grip : grips) {
+    grip->visible = false;
```

Comment:
Any reason you went for adding a member variable instead of using the existing `grip->setVisible(false)`?

I've got a refactor up my sleeve that will avoid the need for this, by the way; I intend to add some flags to `MeshItem` to hide grips and wireframes.

Reply:
setVisible makes them not select-able anymore. I needed to be able to select and move them while they are hidden.

Reply:
Okay, that's what I thought might be the case. I had run into the same issue. Wanted to make sure.